packages/atomic/src/components/common/facets/category-facet/search-value.spec.ts

```diff
@@ -0,0 +1,183 @@
+import {renderFunctionFixture} from '@/vitest-utils/testing-helpers/fixture';
+import {createTestI18n} from '@/vitest-utils/testing-helpers/i18n-utils';
+import {html} from 'lit';
+import {vi, describe, it, expect, beforeAll} from 'vitest';
+import {
+  renderCategoryFacetSearchValue,
+  type CategoryFacetSearchValueProps,
+} from './search-value';
+
+vi.mock('../../../../utils/field-utils', () => ({
+  getFieldValueCaption: vi.fn(
+    (field: string, value: string) => `${field}: ${value}`
+  ),
+}));
+
+describe('renderCategoryFacetSearchValue', () => {
+  let i18n: Awaited<ReturnType<typeof createTestI18n>>;
+
+  beforeAll(async () => {
+    i18n = await createTestI18n();
+  });
+
+  const renderComponent = async (
+    props: Partial<CategoryFacetSearchValueProps> = {}
+  ) => {
+    const defaultProps: CategoryFacetSearchValueProps = {
+      value: {
+        count: 42,
+        path: ['electronics', 'computers'],
+        displayValue: 'laptops',
+      },
+      i18n,
+      field: 'category',
+      onClick: vi.fn(),
+      searchQuery: 'laptop',
+    };
+    const mergedProps = {...defaultProps, ...props};
+    const container = await renderFunctionFixture(
+      html`${renderCategoryFacetSearchValue({props: mergedProps})}`
+    );
+
+    return {
+      container,
+      li: container.querySelector('li'),
+      button: container.querySelector('button'),
+      labelSpan: container.querySelector('span[part="value-label"]'),
+      countSpan: container.querySelector('.value-count'),
+      countSpanByPart: container.querySelector('[part="value-count"]'),
+      pathContainer: container.querySelector('[part="search-result-path"]'),
+      separators: container.querySelectorAll('.mx-0\\.5'),
+      truncateSpans: container.querySelectorAll('.truncate, .max-w-max'),
+    };
+  };
+
+  it('should render a list item element', async () => {
+    const {li} = await renderComponent();
+
+    expect(li).toBeInTheDocument();
+  });
+
+  it('should render button with correct structure', async () => {
+    const {button} = await renderComponent();
+
+    expect(button).toBeInTheDocument();
+  });
+
+  it('should apply search-result part to button', async () => {
+    const {button} = await renderComponent();
+
+    expect(button).toHaveAttribute('part', 'search-result');
+  });
+
+  it('should render facet value label highlight', async () => {
+    const {labelSpan} = await renderComponent();
+
+    expect(labelSpan).toBeInTheDocument();
+  });
+
+  it('should render count with correct formatting', async () => {
+    const {countSpan} = await renderComponent();
+
+    expect(countSpan).toHaveTextContent('(42)');
```

Comment:
This could be combined with 'should format large numbers correctly', e.g., by using 1234 as a value instead of 42.